autopts/wid/bap.py

```diff
@@ -305,12 +317,18 @@ def hdl_wid_114(params: WIDParams):
 
     presentation_delay = 40000
     subgroups = 1
-    broadcast_id = btp.bap_broadcast_source_setup(
-        streams_per_subgroup, subgroups, coding_format, vid, cid,
-        codec_ltvs_bytes, *qos_config, presentation_delay)
 
-    stack = get_stack()
-    stack.bap.broadcast_id = broadcast_id
+    iut = get_iut()
+
+    if iut.name == "Zephyr":
+        log(f"Zephyr: bap_broadcast_source_setup_v2")
```

Comment:
@sjanc How much effort would it be to implement reading and using `BTP_BAP_READ_SUPPORTED_COMMANDS` instead of checking the iut.name here?

Reply:
don't know, probably not that much (even if one would do it in generic way for all services)